apps/builder/app/routes/leak.ts

```diff
@@ -0,0 +1,20 @@
+import { json, type LoaderFunction } from "@remix-run/server-runtime";
```

Comment:
could you clarify what this route for?

Reply:
leak testing, will remove

Reply:
The idea that if you will be able to get information of the visitor using this links it will be a security bug

Reply:
needs comment with repro steps

Reply:
For now no repro, we need to try to find

Reply:
It's like endpoint without any permissions check and even without it should not be possible (except copy-paste bad actor code to somehow see other user data)

Reply:
see `message: "If you see other user then data leaked",`